bin/persephone.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-java -server -jar cli.jar $@
+MACHINE=`bin/machine.sh`
+
+runtimes/$MACHINE/bin/java -server -jar cli.jar $@
```

Comment:
So basically my question during standup: will using the bundled instead of local JRE cause a breaking change?

Reply:
As long as you are not actually publishing a bundle for download then I see no issue and its non-breaking.